### DISCUSSION TITLE
configuration parameters for fastest responses

### DISCUSSION THREAD

**Discussion by utanapishtim** [deepgram/community#468] [General help] [answered]
Hello! I have been experimenting with different models and endpointing parameters to find the fastest and most accurate streaming STT results for spoken English (e.g. in a video call), and I was curious if there is any internal benchmarks or parameter configurations that are known to work well (even given mmv)?

**Comment by team-deepgram** [comment]
Thanks for asking your question about Deepgram! If you didn't already include it in your post, please be sure to add as much detail as possible so we can assist you efficiently, such as:
- The `request_id` if you have a question about your requests or transcription responses.
- The features you used or the full `api.deepgram.com` URL you sent your request to, including parameters.
- Any code snippets you can share.

**Comment by jkroll-deepgram** [comment, marked as answer]
Hi @utanapishtim, are you looking specifically for recommendations for minimizing Deepgram's streaming latency? Can you share more about your use case and architecture? One doc to reference is our [Best Practices for Utterance Segmentation](https://developers.deepgram.com/docs/understand-endpointing-interim-results#best-practices-for-utterance-segmentation), if your goal is to detect when a user has completed an utterance.

Another option to consider if you need the lowest possible latency in streaming audio is that we offer [on-premises deployment](https://developers.deepgram.com/docs/on-prem-introduction), where you can run Deepgram's streaming models on your own hardware, for enterprise-level usage (with a yearly commitment). Our hosted endpoint typically has latency of ~500-800ms, whereas on-prem can reach ~300-500ms.
